### PR TITLE
Remove middleman from presskit.

### PR DESCRIPTION
**Pull request by brendastorer**
Middleman instance was 2 years old and either needed updating or removal. Since this kit only gets updated about once every two years and middleman was only being used for Sass compiliation, I stripped it down to a static version that is simpler and should be easier to maintain. Everything looks and acts the same as before (for now). HTML Markup and Sass is the same (for now).

- Simpler GitHub structure: no longer requires 3 separate branches to host on GitHub pages. This can live in `master` and `gh-pages` only. We can delete the `middleman-source` branch

- Updated Bourbon to 5.0.0. Updated Bitters. Removed Neat (just didn't need it for the single page with 2 columns)

- Removed Philadelphia from office list. Updated copyright year to current year. Resolves #15 

- Uses `sass --watch` to compile Sass

- alt descriptions on images

### CODE REVIEW

File: images/download-icon.svg [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (ohaicristina) [inline images/download-icon.svg]: Run svgo on these svgs?

File: index.html [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+    <link href="stylesheets/base.css" rel="stylesheet" type="text/css" />
+    <script src="//use.typekit.net/gnk7llu.js"></script>
+    <script>try{Typekit.load();}catch(e){}</script>
+    <script src="http://ajax.googleapis.com/ajax/libs/jquery/1.11.1/jquery.min.js"></script>
+  </head>
+  <body class="index">
+    <div class="page-wrap">
+      <header>
+        <div class="main-content-wrap">
+          <h1 class="page-heading">Brand Assets & Guidelines</h1>
```
Comment (ohaicristina) [inline index.html]: Should that ampersand be `&amp;`?

File: index.html [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+        </nav>
+      </div>
+      <div class="main-content-wrap">
+        <section class="intro">
+          <p>The purpose of this guide is to ensure that consistency is maintained across the thoughtbot brand, whether assets are used internally or externally. Please reference this guide whenever you use these assets to ensure that they are used correctly.</p>
+          <a class="download-assets" href="assets/thoughtbot-assets-pack.zip">Download Assets Pack</a>
+        </section>
+        <section id="signature">
+          <h2>The Signature</h2>
+          <p>The thoughtbot logo signature combines the two most important visual elements of our brand, the thoughtbot logotype and the thoughtbot symbol (Ralph). The examples below illustrate the recognizable graphic features that make the thoughtbot signature unique.</p>
```
Comment (ohaicristina) [inline index.html]: Might be good to wrap these p tags so they're easier to read

File: stylesheets/base.css [hunk 4]
```diff
@@ -0,0 +1,400 @@
...
+  vertical-align: middle;
+  white-space: nowrap; }
+  button:hover, button:focus, [type='button']:hover, [type='button']:focus, [type='reset']:hover, [type='reset']:focus, [type='submit']:hover, [type='submit']:focus {
+    background-color: #9e2426;
+    color: #fff; }
+  button:disabled, [type='button']:disabled, [type='reset']:disabled, [type='submit']:disabled {
+    cursor: not-allowed;
+    opacity: 0.5; }
+    button:disabled:hover, [type='button']:disabled:hover, [type='reset']:disabled:hover, [type='submit']:disabled:hover {
+      background-color: #c52d2f; }
```
Comment (ohaicristina) [inline stylesheets/base.css]: huh, I know this is plain css, but I'm wondering what the benefit to having the closing bracket on the same line as the last attribute is?

File: stylesheets/sass/_layout.scss [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+  width: 20%;
+}
+
+.main-content-wrap {
+  margin-left: 22%;
+  width: 78%;
+}
+
+section {
+  margin-bottom: 130px;
```
Comment (ohaicristina) [inline stylesheets/sass/_layout.scss]: Is it worth making variables for any of these spacing values that are repeated?

File: stylesheets/base.css (hunk 4 above)
Comment (brendastorer) [inline stylesheets/base.css]: This is what the Sass compiles to, so check out the Sass folder.

File: stylesheets/sass/_main.scss [hunk 6]
```diff
@@ -0,0 +1,241 @@
...
+  font-family: $heading-font-family;
+  padding: 10px;
+  position: relative;
+  text-decoration: none;
+
+  &.active {
+    color: $thoughtbot-red;
+
+    &::before {
+      border-bottom: 8px solid transparent;
```
Comment (ohaicristina) [inline stylesheets/sass/_main.scss]: same variable question for the borders

File: stylesheets/sass/_main.scss [hunk 7]
```diff
@@ -0,0 +1,241 @@
...
+    margin-top: 50px;
+
+    .color-specs {
+      display: inline-block;
+      vertical-align: top;
+
+      h3,
+      h4,
+      hr {
+        margin: 0 0 20px 22px;
```
Comment (ohaicristina) [inline stylesheets/sass/_main.scss]: margin mixin?

File: stylesheets/base.css (hunk 4 above)
Comment (ohaicristina) [inline stylesheets/base.css]: ahh! I see now. Don't mind me...

**Comment by ohaicristina** [comment]
😎  Nice!
+1 to moving away from Middleman. Feels like we were just using it for the sake of using it. 
Only thing is now we've got all those bourbon/bitters files locally, which I think is fine considering how simple this page needs to be.

**Comment by brendastorer** [comment]
Thanks @ohaicristina! A few of your Sass suggestions I'm going to wait on and put in play when updating the styles to new branding in a different branch.

File: README.md [hunk 8]
```diff
@@ -58,7 +57,12 @@ Social
 Copyright
 ---------
 
-Copyright 2015 thoughtbot, inc.
+Copyright 2016 thoughtbot, inc.
```
Comment (tysongach) [inline README.md]: We should add `Closes #15` to the commit message, so to reference and auto-close [that PR](https://github.com/thoughtbot/presskit/pull/15).

File: index.html [hunk 9]
```diff
@@ -0,0 +1,166 @@
+<!doctype html>
+<html>
```
Comment (tysongach) [inline index.html]: Can we add `lang="en"` to improve a11y?

File: index.html [hunk 10]
```diff
@@ -0,0 +1,166 @@
...
+  <head>
+    <meta charset="utf-8">
+    <meta content="IE=edge,chrome=1" http-equiv="X-UA-Compatible">
+    <title>thoughtbot Brand Assets & Guidelines</title>
+    <link href="stylesheets/normalize.css" rel="stylesheet" type="text/css" />
```
Comment (tysongach) [inline index.html]: Instead of importing normalize.css on its own, can we `@import` via Sass into `base.css`?
